registrations/models.py

```diff
@@ -67,40 +67,53 @@ def formfield(self, **kwargs):
         )
 
 
-class Registration(models.Model):
-    event = models.OneToOneField(
-        Event,
-        on_delete=models.CASCADE,
-        related_name="registration",
-        null=False,
+class CreatedModifiedBaseModel(models.Model):
+    created_at = models.DateTimeField(
+        verbose_name=_("Created at"),
+        null=True,
         blank=True,
+        auto_now_add=True,
     )
-    attendee_registration = models.BooleanField(default=False, null=False)
-    audience_min_age = models.PositiveSmallIntegerField(
-        verbose_name=_("Minimum recommended age"), blank=True, null=True, db_index=True
-    )
-    audience_max_age = models.PositiveSmallIntegerField(
-        verbose_name=_("Maximum recommended age"), blank=True, null=True, db_index=True
-    )
-
-    created_at = models.DateTimeField(verbose_name=_("Created at"), auto_now_add=True)
     last_modified_at = models.DateTimeField(
-        verbose_name=_("Modified at"), null=True, blank=True, auto_now=True
+        verbose_name=_("Modified at"),
+        null=True,
```

Comment:
I see this was already nullable but at least I would rather set  it non-nullable and set a default in the migration. Unless there was some good reason to do this originally..  A lot of fields in linkedevents are indeed nullable :sweat_smile:

Reply:
Indeed, some of these would be better off being non-nullable. I think the reasoning for allowing this to be nullable is the same as for the created_at timestamp: the existing data where it having a null valuable is perhaps the most appropriate to not give a false default value for them.

Reply:
False default values should not really matter as the service is not in production yet, so all those nullability things could and should be fixed now IMO.

Reply:
The datetimefields have now been made non-nullable.